### PR TITLE
fix: collector tasks time interval correction during recover

### PR DESCRIPTION
**Pull request by rahulguptajss**

### CODE REVIEW

File: cmd/poller/collector/collector.go [hunk 1]
```diff
@@ -308,27 +308,27 @@ func (me *AbstractCollector) Start(wg *sync.WaitGroup) {
...
 					if !me.Schedule.IsStandBy() {
 						//logger.Error(me.Prefix, err.Error())
-						me.Logger.Warn().Msgf("target unreachable, entering standby mode (retry to connect in %d s)", retryDelay)
+						me.Logger.Warn().Msgf("task %s, target unreachable, entering standby mode (retry to connect in %d s)", task.Name, retryDelay)
 					}
 					me.Logger.Debug().
 						Int("retryDelay", retryDelay).
 						Err(err).
-						Msg("Target unreachable, entering standby mode (retry in retryDelay s)")
+						Msgf("task %s, Target unreachable, entering standby mode (retry in retryDelay s)", task.Name)
```
Comment (cgrinds) [inline cmd/poller/collector/collector.go:316]: Str("task", task.Name)

**Review by cgrinds — DISMISSED** [review]
nice catch!

File: cmd/poller/schedule/schedule.go [hunk 2]
```diff
@@ -104,14 +106,13 @@ func (s *Schedule) IsStandBy() bool {
...
 
-// SetStandByMode intiailzes StandbyMode: Schedule will suspend all tasks until
+// SetStandByMode initializes StandbyMode: Schedule will suspend all tasks until
 // the critical task t has succeeded. The temporary interval i will be used for
 // the task until Schedule recovers to normal mode.
 func (s *Schedule) SetStandByMode(t *task, i time.Duration) {
 	for _, x := range s.tasks {
 		if x.Name == t.Name {
 			s.standByTask = t
-			s.cachedInterval = t.interval // remember normal interval of task
```
Comment (Hardikl) [inline cmd/poller/schedule/schedule.go:114]: don't we need to cache this previous interval?

Comment (cgrinds) [reply cmd/poller/schedule/schedule.go:114]: the cache is created when the task is added to the schedule and then the cached value can never change

File: cmd/poller/collector/collector.go (hunk 1 above)
Comment (rahulguptajss) [reply cmd/poller/collector/collector.go]: done.

File: cmd/poller/collector/collector.go [hunk 3]
```diff
@@ -308,27 +308,27 @@ func (me *AbstractCollector) Start(wg *sync.WaitGroup) {
 					}
 					if !me.Schedule.IsStandBy() {
 						//logger.Error(me.Prefix, err.Error())
-						me.Logger.Warn().Msgf("target unreachable, entering standby mode (retry to connect in %d s)", retryDelay)
+						me.Logger.Warn().Msgf("task %s, target unreachable, entering standby mode (retry to connect in %d s)", task.Name, retryDelay)
```
[the remark this answers is not in the document]
Comment (rahulguptajss) [reply cmd/poller/collector/collector.go]: done

File: cmd/poller/collector/collector.go [hunk 4]
```diff
@@ -308,27 +308,28 @@ func (me *AbstractCollector) Start(wg *sync.WaitGroup) {
 					}
 					if !me.Schedule.IsStandBy() {
 						//logger.Error(me.Prefix, err.Error())
-						me.Logger.Warn().Msgf("task %s, target unreachable, entering standby mode (retry to connect in %d s)", task.Name, retryDelay)
+						me.Logger.Warn().Str("task", task.Name).Msgf("target unreachable, entering standby mode (retry to connect in %d s)", retryDelay)
```
Comment (cgrinds) [inline cmd/poller/collector/collector.go]: Int("retryDelay", retryDelay) and then change msgf to msg

**Review by cgrinds — APPROVED** [review]

**Pull request merged**